pkg/plugins/golang/v4/scaffolds/internal/templates/webhooks/webhook_suitetest.go

```diff
@@ -229,8 +229,18 @@ var _ = BeforeSuite(func() {
 
 	ctx, cancel = context.WithCancel(context.TODO())
 
+	var err error
+	err = %s.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
+
+	err = %s.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
+
+	%s
+
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
+		CRDInstallOptions:     envtest.CRDInstallOptions{Scheme: scheme.Scheme},
```

Comment:
made this change because of this https://github.com/kubernetes-sigs/controller-runtime/issues/1882#issuecomment-1138880376